app.rb

```diff
@@ -0,0 +1,146 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'books'
+require_relative 'rentals'
+
+class App
+  private
+
+  def create_student(arr_person)
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permision? [Y/N]: '
+    get_permision = gets.chomp.upcase
+    until %w[Y N].include?(get_permision)
+      puts 'Please Enter valid permision'
+      get_permision = gets.chomp.upcase
```

Comment:
- [ ] Your app is breaking in an attempt to create a person of type student. When I enter an option (`y`) for has permission your app then throws an error. Kindly ensure to improve upon this aspect.

**Creating student person error:**
 ![image](https://user-images.githubusercontent.com/12745474/211175586-1f70e0d4-9ce4-4a01-8271-2e1caf3c46b4.png)